src/list.rb

```diff
@@ -131,7 +131,7 @@ def purge_dir(dir)
   list_cache = cache.get 'list'
 
   if list_cache.nil?
-    uri = URI "#{base_uri}/teams/#{team_id}/entries"
+    uri = URI "#{base_uri}/entries"
```

Comment:
@lkngtn do we want to be able to filter by team?

Reply:
If a user is a member of multiple teams it might be useful, though if that were the case it would need to be a parameter "idtlist team". 

Not sure if its worth the extra complexity or not, especially if we are still limited to 25 entries. The use case I would see for filtering by team is for different context, IE you work for multiple teams and have different goals for each, you may not work in a specific context every week. So it would be nice to filter out only the stuff you are currently working on, the problem with only the 25 most recent you probably wont see all your goals for context that you havent worked in recently.

Reply:
Good point.